examples/cd_distillation_cifar10.ipynb

```diff
@@ -0,0 +1,781 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_labels = ['0', '1', '2', '3', '4', '5']\n",
+    "\n",
+    "N = 6\n",
+    "ind = np.arange(N)\n",
+    "width = .35\n",
+    "\n",
+    "fig_bar_cd, ax = plt.subplots()\n",
+    "ax2 = ax.twinx()\n",
+    "\n",
+    "p0 = ax.bar(ind[0], score_x.mean(), yerr=score_x.std(), capsize=2)\n",
+    "p1 = ax.bar(ind[1:], mu_noharm, width, yerr=std_noharm, capsize=2)\n",
+    "p2 = ax.bar(ind[1:] + width, mu_harm, width, yerr=std_harm, capsize=2)\n",
+    "\n",
+    "ax.set_title('Harmfullness Scores and Accuracy by Corruption Severity')\n",
+    "ax.set_xticks(ind + width / 2)\n",
+    "ax.set_xticklabels(plot_labels)\n",
+    "ax.set_ylim((-2))\n",
+    "ax.legend((p1[0], p2[0]), ('Not Harmful', 'Harmful'), loc='upper right', ncol=2)\n",
+    "ax.set_ylabel('Score')\n",
+    "ax.set_xlabel('Corruption Severity')\n",
+    "\n",
+    "color = 'tab:red'\n",
+    "ax2.set_ylabel('Accuracy', color=color)\n",
+    "ax2.plot(acc, color=color)\n",
+    "ax2.tick_params(axis='y', labelcolor=color)\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Plot p-values for contaminated batches\n",
+    "\n",
+    "In order to simulate a realistic scenario, we perform a K-S test on batches of instance which are increasingly contaminated with corrupted data. The following steps are implemented:\n",
+    "\n",
+    "* We randomly pick `n_ref` samples from the non-currupted test set to be used as a reference set in the initialization of the K-S drift detector.  \n",
+    "\n",
+    "* We sample batches of data of size `batch_size` contaminated with an increasing number of harmful corrupted data and harmless corrupted data. \n",
+    "\n",
+    "* We apply the K-S test to these batches and obtain the p-values for each of them. \n",
```

Comment:
"We apply the K-S test to these batches and obtain the p-values for each of them."
Maybe change to:
"The K-S detector predicts whether drift occurs between the contaminated batches and the reference data and returns the p-values of the test."